hypervisor/bsp/firmware_sbl.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2019 Intel Corporation. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/* this is for both SBL and ABL platform */
+
+#include <hypervisor.h>
+#include <firmware_sbl.h>
+
+static void sbl_init(void)
+{
+	/* nothing to do for now */
+}
+
+static uint64_t sbl_get_ap_trampoline(void)
+{
+	return e820_alloc_low_memory(CONFIG_LOW_RAM_SIZE);
+}
+
+static void* sbl_get_rsdp(void)
```

Comment:
Will it be called?  If we return NULL here at runtime, we are trouble ?

Reply:
Yes, it'll be called and it's allowed to return NULL, because we already handled the NULL pointer [here](https://github.com/projectacrn/acrn-hypervisor/blob/master/hypervisor/boot/acpi.c#L150).